pyam/core.py

```diff
@@ -148,6 +148,11 @@ def tail(self, *args, **kwargs):
         """Identical to pd.DataFrame.tail() operating on data"""
         return self.data.tail(*args, **kwargs)
 
+    @property
+    def empty(self):
+        """Indicator whether ``IamDataFrame`` is empty"""
```

Comment:
```suggestion
        """Is the ``IamDataFrame`` empty?"""
```

Reply:
I was using the pandas docstring as a template, see https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.empty.html